src/main/java/com/google/codeu/utilities/MessageUtil.java

```diff
@@ -48,4 +65,16 @@ public static String formatImages(String message){
 
     return textWithImages;
   }
+
+  /*
+   * This method takes the message entered by the user and replaces markdown with
+   * html tags in the html of the page using the regular expressions above (regex)
+   */
+  public static String formatText(String message) {
+	  String bold = message.replaceAll(BOLD_REGEX, BOLD_REPLACEMENT);
```

Comment:
Rather than creating a separate variable for each, you can just chain all the calls:

return message.replaceAll(BOLD_REGEX, BOLD_REPLACEMENT)
    .replaceAll(ITALIC_REGEX, ITALIC_REPLACEMENT)
    .replaceAll(...etc...)